app/src/main/res/menu/token_info_menu.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/token_info_menu_delete"
+        android:icon="@drawable/ic_delete_toolbar_24dp"
+        android:title="@string/delete"
+        app:showAsAction="ifRoom" />
```

Comment:
I would never show that as a action. To hide the delete option more, maybe we should check with michel